spectacles/validators/sql.py

```diff
@@ -169,11 +178,23 @@ def _create_explore_test(
         query = self.client.create_query(
             explore.model_name, explore.name, dimensions, fields=["id", "share_url"]
         )
+
+        # Create separate chunked queries for execution, we don't store compiled SQL
+        # or the Explore URL for these queries
+        chunk_queries: List[Query] = []
+        for chunk in chunks(dimensions, size=CHUNK_SIZE):
+            query = self.client.create_query(
+                explore.model_name, explore.name, chunk, fields=["id", "share_url"]
+            )
+            chunk_queries.append(Query(query["id"]))
+
+        sql = self.client.run_query(query["id"]) if compile_sql else None
```

Comment:
This client function feels like it could be named a bit better, given it doesn't actually run the query, as far as I can tell.

Reply:
I've kept the names of the client methods 1:1 with the Looker API endpoints for simplicity. You do actually use the Run Query endpoint with the `sql` format to compile.